### PR TITLE
Create bower package

### PR DESCRIPTION
**Pull request by rbecheras**

### CODE REVIEW

**Comment by rbecheras** [comment]
I tagged the version 1.0.0 and set it in bower.json too

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "eventdispatcher.js",
+  "version": "1.0.0",
+  "authors": [
+    "Rémi Becheras <rbecheras@sirap.fr>"
```
Comment (mrdoob) [inline bower.json]: You assigned yourself as the author?

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "description": "JavaScript events for custom objects",
+  "main": "src/EventDispatcher.js",
+  "moduleType": [
+    "globals"
+  ],
+  "keywords": [
+    "event"
+  ],
+  "license": "MIT",
+  "homepage": "http://gitlab.sirap.fr/open-source/eventdispatcher.js"
```
Comment (mrdoob) [inline bower.json]: What site is this?

File: bower.json (hunk 1 above)
Comment (rbecheras) [inline bower.json]: Oh yes I'm sorry I worked on a fork in a private gitlab server and I generated the bower package from it so my name an my private repo was put in. I'll fix that soon

File: bower.json (hunk 2 above)
Comment (rbecheras) [inline bower.json]: It's my company private gitlab server. I'll fix it too

**Comment by rbecheras** [comment]
@mrdoob , it's done

**Comment by rbecheras** [comment]
I registered the bower component to bower public register:

```
$ bower info eventdispatcher.js
bower eventdispatcher.js#*  not-cached https://github.com/mrdoob/eventdispatcher.js.git#*
bower eventdispatcher.js#*     resolve https://github.com/mrdoob/eventdispatcher.js.git#*
bower eventdispatcher.js#*    checkout master
bower eventdispatcher.js#*     invalid-meta eventdispatcher.js is missing "main" entry in bower.json
bower eventdispatcher.js#*     invalid-meta eventdispatcher.js is missing "ignore" entry in bower.json
bower eventdispatcher.js#*         resolved https://github.com/mrdoob/eventdispatcher.js.git#57d849444a

{
  name: 'eventdispatcher.js',
  homepage: 'https://github.com/mrdoob/eventdispatcher.js'
}

No versions available.

```

However the command says "No versions available." still 

1. you merge the PR 
2. and THEN you push a new git tag: 

    $ git pull <remote> master
    $ git tag v1.0.0
    $ git push <remote> 
    $ git push <remote> --tags

**Comment by mrdoob** [comment]
Done!

**Pull request merged**